src/internal/components/pluggableVisualizations/comboChart/tests/PluggableComboChart.spec.tsx

```diff
@@ -382,4 +382,116 @@ describe("PluggableComboChart", () => {
             },
         );
     });
+
+    describe("show in percent", () => {
+        it("should keep percentage config if there is only one master measure", async () => {
+            const refPointMock: IReferencePoint = referencePointMocks.measureWithDerivedAndPercentageRefPoint;
+            const extRefPoint: IExtendedReferencePoint = await getExtendedReferencePoint(refPointMock);
+
+            expect(extRefPoint.buckets).toEqual([
+                {
+                    ...primaryMeasureBucketProps,
+                    items: [
+                        {
+                            ...referencePointMocks.masterMeasureItems[0],
+                            showOnSecondaryAxis: false,
+                            showInPercent: true,
+                        },
+                        {
+                            ...referencePointMocks.derivedMeasureItems[0],
+                            showOnSecondaryAxis: false,
+                            showInPercent: true,
+                        },
+                    ],
+                },
+                secondaryMeasureBucketProps,
+                refPointMock.buckets[1],
+            ]);
+        });
+
+        it("should ignore percentage config if there are multiple master measures on primary measure bucket", async () => {
+            const refPointMock: IReferencePoint = referencePointMocks.multipleMeasuresWithPercentageRefPoint;
+            const extRefPoint: IExtendedReferencePoint = await getExtendedReferencePoint(refPointMock);
+
+            expect(extRefPoint.buckets).toEqual([
+                {
+                    ...primaryMeasureBucketProps,
+                    items: [
+                        {
+                            ...referencePointMocks.masterMeasuresWithPercentage[0],
+                            showOnSecondaryAxis: false,
+                            showInPercent: null,
```

Comment:
I would expect here `true / false` for boolean value ... why its there received `null`

Reply:
Because of this function: https://github.com/gooddata/gooddata-react-components/blob/next-7.1.0/src/internal/utils/bucketConfig.ts#L72
```
function removeShowInPercent(measure: IBucketItem) {
    return set(measure, SHOW_IN_PERCENT, null);
}
```
Im not sure about the author's intent, but I guess that it's set to original value same as it was created (https://github.com/gooddata/gdc-analytical-designer/blob/develop/app/models/bucket_item.ts#L15)